src/fragments/Markdown.react.js

```diff
@@ -2,12 +2,18 @@ import React, {Component} from 'react';
 import {type} from 'ramda';
 import Markdown from 'react-markdown';
 
+import Highlight from '../utils/Highlight';
 import {propTypes, defaultProps} from '../components/Markdown.react';
 import '../components/css/highlight.css';
 
 export default class DashMarkdown extends Component {
     constructor(props) {
         super(props);
+        /* eslint-disable no-new */
```

Comment:
Disabling this error as we really only do want the side-effects of creating a new `Highlight` object. An alternative is wrapping all of the `Highlight` code with a function instead of a class.

Reply:
Can you explain what's happening here?